### PR TITLE
#8082 bug: refactoring stats card fact card

### PR DESCRIPTION
**Pull request by DipakShrestha18**

### CODE REVIEW

**Review by chriddell — CHANGES_REQUESTED** [review]
Code looks good, but... can we name the FE component `FactCard` and drop the `WithHyperlink`? Reasons being...

- we do not have two variants `FactCard` and `FactCardWithHyperlink`
- `WithHyperlink` is fairly meaningless (the term "Hyperlink" is deprecated and doesn't tend to be used any more)
- the relationship between `FactCard` and `fact_card_with_hyperlink` is made explicit elsewhere in the code, so people are free to use either term and the relationship is visible
- it is what I proposed here https://github.com/wellcometrust/corporate/issues/8082#issuecomment-774061301 (this is the least important, really)

**Review by chriddell — COMMENTED** [review]
Sorry, another annoying point... the branch name should have a ticket number on the end. Also, just a thought - I don't think this is a `bug` branch, nothing is broken (it's just that we use the wrong names).

I think it should be a feature, so

`feature/refactoring-statistic-card-to-fact-card-#8082`

File: src/components/FactCard/FactCard.tsx [hunk 1]
```diff
@@ -13,29 +13,26 @@ type StatisticCardProps = {
   titleAs?: 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
 };
 
-export const StatisticCard = ({
+export const FactCardProps = ({
```
Comment (creido-welly) [inline src/components/FactCard/FactCard.tsx]: You have duplication of the type `FactCardProps` here.  I'm guessing this is a find and replace oversight. It should be `FactCard`.

File: src/components/FactCard/FactCard.tsx [hunk 2]
```diff
@@ -54,4 +51,4 @@ export const StatisticCard = ({
   );
 };
 
-export default StatisticCard;
+export default FactCardProps;
```
Comment (creido-welly) [inline src/components/FactCard/FactCard.tsx]: See comment on line 16 - default export should be `FactCard` to match

File: src/components/FactCard/FactCard.tsx (hunk 1 above)
Comment (DipakShrestha18) [reply src/components/FactCard/FactCard.tsx]: You're right find and replace. 

Re-review please.

**Review by creido-welly — APPROVED** [review]

**Comment by DipakShrestha18** [comment]
@chriddell Not sure what's your CR is in this. It's not allowing me to merge this PR.

**Comment by chriddell** [comment]
This is mostly fine now, though the branch name should have the ticket number on the end.

Can you check out a new branch (or rename this one - not sure if that's actually possible...):

```feature/refactoring-statistic-card-fact-card-#<ticketNumber>```

(I wouldn't consider this a bug - it's a refactoring but nothing is broken)

**Pull request closed without merging**